streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -1057,6 +1063,69 @@ public void shouldUpdateStandbyTask() throws Exception {
         assertEquals(0, thread.standbyRecords().size());
     }
 
+    @Test
+    public void shouldCreateStandbyTask() {
+        final MockProcessor mockProcessor = new MockProcessor();
```

Comment:
We could reuse initialization code for L1068 - L1070

Reply:
Ack